tests/utils.py

```diff
@@ -2,6 +2,22 @@
 Utilities for testing.
 """
 
+import os
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+
+import pytest
+import synapseclient
+import synapseutils
```

Comment:
are these now dependencies of `nViz`? Is there a way to make them only optional testing dependencies?